src/reconcile/rec_write.c

```diff
@@ -2637,8 +2597,31 @@ __rec_split_write(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *chunk
         /* Turn off compression adjustment for delta. */
         compressed_size = 0;
 
+        delta_pct = (r->delta.size * 100) / chunk->image.size;
         if (F_ISSET(r->ref, WT_REF_FLAG_INTERNAL)) {
             WT_STAT_CONN_DSRC_INCR(session, rec_page_delta_internal);
+
+            /*
+             * If we decide to write the delta we packed, track the number of bytes saved by
+             * avoiding writing the full page image.
+             *
+             * Also track how large the delta is compared to the full page image.
+             */
+            WT_STAT_CONN_INCRV(session, block_byte_write_saved_delta_intl, chunk->image.size);
```

Comment:
shouldnt this be  chunk->image.size - delta size?

Reply:
When I think about the 'saved' stat, it's essentially showing that we wrote 2KB (using deltas) instead of 20KB (writing full page images), so we saved 18KB. Over time, this stat accumulates. For instance, it might show 720MB saved, which I interpret as: if we had been writing full page images instead of deltas, we would have written 720MB more.